pkg/services/alerting/test_rule.go

```diff
@@ -50,7 +51,7 @@ func handleAlertTestCommand(cmd *AlertTestCommand) error {
 func testAlertRule(rule *Rule) *EvalContext {
 	handler := NewEvalHandler()
 
-	context := NewEvalContext(context.Background(), rule)
+	context := NewEvalContext(context.Background(), rule, &setting.Cfg{})
```

Comment:
Not sure if this is for testing only and whether a real Cfg needs to be propagated here.